packages/next/build/swc/options.js

```diff
@@ -79,7 +79,8 @@ function getBaseSWCOptions({
           importSource:
             jsConfig?.compilerOptions?.jsxImportSource ??
             (nextConfig?.compiler?.emotion ? '@emotion/react' : 'react'),
-          runtime: 'automatic',
+          runtime:
+            nextConfig?.compiler?.emotion?.transformRuntime || 'automatic',
```

Comment:
Should this config be nested under the `emotion` config as it's not specific just to `emotion`? 

Seems like it'd be more fitting as a separate field e.g. `nextConfig.compiler.react.runtime`

Reply:
I think so, and IIRC, we already have a value for swc.
But it's not configurable.